Makefile

```diff
@@ -69,6 +69,21 @@ centos6: make-rpmbuild centos
 centos-7: centos7
 centos7: make-rpmbuild centos
 
+# Erlang is built from source on ARMv8
+# These packages are not installed, but the files are present
+armv8-prep:
```

Comment:
Change the name of this to `drop-deb-deps-for-source-arch` please, as I anticipate we'll need to do this for `ppc64le` as well.